blog/mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 INHERIT: config/nav.yml
 
 site_name: Knative
-site_url: https://knative.dev/blog
+site_url: https://knative.dev/docs
```

Comment:
As noted in earlier comment, think we need to use `extra.homepage` instead due to the fact that we're using the `mike` utility

Reply:
Ok, I made an extra commit and will check the build. I am happy to close this PR though and let you send the fix so that you are the author.

Reply:
Not necessary -- it's your PR, all I did was review it